external-resolver/src/test/scala/org/genivi/sota/resolver/test/Requests.scala

```diff
@@ -131,6 +131,10 @@ trait PackageRequestsHttp {
   def getAffected(namespace: Namespace, ids: Set[PackageId])
                  (implicit ec: ExecutionContext): HttpRequest =
     Post(Resource.uri("packages", "affected").withQuery(Query("namespace" -> namespace.get)), ids)
+
+  def getPackageStats(namespace: Namespace, name: PackageId.Name)
```

Comment:
These tests need a cleanup. Some still put namespace in query parameter. Let's do later.